core/src/trezor/ui/model/button/__init__.py

```diff
@@ -0,0 +1,13 @@
+from .. import IS_T1
+
+if False:
+    from typing import TYPE_CHECKING
+else:
+    TYPE_CHECKING = False
+
+if IS_T1:
+    from .t1 import *  # noqa: F401,F403
+
+# FIXME: without the condition mypy complains about Incompatible import
+elif not TYPE_CHECKING:
+    from .tt import *  # noqa: F401,F403
```

Comment:
I could get away without `TYPE_CHECKING` in the other `__init__.py` files. Here, if removed, mypy visits both `if` branches and complains about importing incompatible types. Both types are classes derived from common base class but I don't know how to tell this to mypy.